Vizzuality/widget-editor
# Standardise the look of the inputs and buttons

This PR standardises the styles of the inputs and buttons, and fixes a couple of minor issues such as z-indexes.

<p align="center">
<img width="661" alt="Screenshot of the filters" src="https://user-images.githubusercontent.com/6073968/96424539-bb799f80-11f2-11eb-8a87-cd0e2298b434.png">
</p>

## Testing instructions

Here's a list of a few items to check:
- Check that the map's boundaries select is now a checkbox
- Check that the checkbox behaves as intented
- Check that the text, number and date inputs have a hover/active style
- Check that the buttons have a hover/active style
- Check that the labels are connected to the inputs (clicking on the labels must put the focus on the inputs)
- Check that the slider handles are displayed below the list of the selects

## Pivotal Tracker

- [Task](https://www.pivotaltracker.com/story/show/175044722) to fix the height of the inputs to 45px
- [Task](https://www.pivotaltracker.com/story/show/173687362) to replace the boundaries select by a checkbox
- [Task 1](https://www.pivotaltracker.com/story/show/172950863) and [task 2](https://www.pivotaltracker.com/story/show/173900644) to fix the z-index of the sliders
